joblib/pool.py

```diff
@@ -196,8 +190,11 @@ def __init__(self, max_nbytes, temp_folder, mmap_mode, verbose=0,
         self._temp_folder = temp_folder
         self._mmap_mode = mmap_mode
         self.verbose = int(verbose)
-        self._context_id = context_id
         self._prewarm = prewarm
+        if context_id is not None:
+            warnings.warn('context_id is deprecated and ignored in joblib'
+                          ' 0.9.4 and will be removed in 0.11',
+                          DeprecationWarning)
```

Comment:
I agree but I would rather not make this code change in the bugfix release to keep the diff readable.